eng/pipelines/docindex.yml

```diff
@@ -75,6 +103,7 @@ jobs:
           TargetRepoOwner: $(DocRepoOwner)
           WorkingDirectory: $(DocRepoLocation)
           ScriptDirectory: $(Build.SourcesDirectory)/eng/common/scripts
+          PushArgs: -f
```

Comment:
Are we certain we want to force push now?

Reply:
We might have nightly pipeline pushes commits during docindex run, so it complains about the commits behind sometime.

By force pushing the commit, we might lose one or two commits from nightly pipeline. The miss on the commits is acceptable to me as following reasons.  First of all, it doesn't occur very often. Second, it doesn't always affect the same package. Last, this is daily refresh docs, so it will show up the right thing in the next day.

Reply:
As long as we don't end up overwriting any manual commits, we might have that I suppose this will be ok.